lib/widgets/message_list.dart

```diff
@@ -1273,7 +1273,7 @@ class MessageWithPossibleSender extends StatelessWidget {
 
     return GestureDetector(
       behavior: HitTestBehavior.translucent,
-      onLongPress: () => showMessageActionSheet(context: context, message: message),
+      onLongPress: () => showMessageActionSheet(context, message: message),
```

Comment:
I guess this one last commit doesn't convince me:
53a24b043 action_sheet [nfc]: Make showMessageActionSheet's context an unnamed param

The existing API there matches that of upstream's `showModalBottomSheet`. I think in turn the idea there is that it's similar in flavor to a widget constructor; and those always take all named arguments.